Header header
string command
string bumperFront
string bumperBack
string emergency
string cutter
string lift
float64 leftSpeed
float64 rightSpeed
float64 outerRight
float64 innerRight
float64 innerLeft
float64 outerLeft
float64 xPos
float64 yPos
float64 heading
